tools/sof-kernel-log-check.sh

```diff
@@ -313,11 +313,19 @@ ignore_str="$ignore_str"'|rt700 sdw:.:.*:700:.: Bus clash detected'
 
 # confirm begin_timestamp is in UNIX timestamp format, otherwise search full log
 journalctlflag="-k -q --no-pager --utc --output=short-monotonic --no-hostname"
-[[ $begin_timestamp =~ ^[0-9]{10} ]] && cmd="journalctl $journalctlflag --since=@$begin_timestamp" || cmd="journalctl $journalctlflag"
+if [[ $begin_timestamp =~ ^[0-9]{10} ]]; then
+    cmd="journalctl $journalctlflag --since=@$begin_timestamp"
+else
+    cmd="journalctl $journalctlflag"
+fi
```

Comment:
This could be
```shell
cmd="journalctl $journalctlflag"
if [[ $begin_timestamp =~ ^[0-9]{10} ]]; then
   cmd="$cmd --since=@$begin_timestamp"
fi
```